russstation/code/game/objects/items/cards_ids.dm

```diff
@@ -0,0 +1,14 @@
+// for some reason called at round start, so let's grab this ref
+/obj/item/card/id/captains_spare/GetID()
+	if(GLOB.egalitarian_mode_active)
+		GLOB.captains_spare = src
+	return ..()
+
+/obj/item/card/id/captains_spare/proc/anti_tide()
+	say("Anti-Tide mechanism activating...")
+	QDEL_IN(src, 50)
+	return list()
+
+/obj/item/card/id/captains_spare/Destroy()
+	explosion(src, 0, 0, 2, 2)
```

Comment:
This makes it so every spare will explode when deleted, which sounds rather painful for admins and people testing on local servers. Could this be refactored to use a timer calling a proc, instead of overwriting Destroy?